src/B2_build_movie_db.sql

```diff
@@ -169,3 +169,6 @@ CREATE TABLE media_production_company (
     "type"                  VARCHAR(20) NOT NULL,
     PRIMARY KEY (media_id, production_company_id)
 );
+
+-- Add a new column to the person table this is for the name rating.
+ALTER TABLE person ADD COLUMN name_rating DECIMAL(3, 2);
```

Comment:
Please just update the table instead of using the alter statement, we are dropping the tables each time we run the script either way so there is no need to split the table definitions through the script